pkg/dump/dumper.go

```diff
@@ -137,10 +137,6 @@ func (d *logDumper) DumpAllNodes(ctx context.Context, nodes corev1.NodeList, add
 			dumped = append(dumped, node)
 		}
 	}
-	// If none of the special nodes was successfully dumped, there is no value in dumping the rest
```

Comment:
This change allows us to debug clusters that fail to get the control plane instance online.

https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/kops/15974/pull-kops-kubernetes-e2e-cos-gce/1711393302265204736/artifacts/34.16.149.221/kops-configuration.log

https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kops/15974/pull-kops-kubernetes-e2e-cos-gce/1711393302265204736

Reply:
As, I mentioned several times, please create separate PRs.